### PR TITLE
Update travel sdk to 2.0.0

### PR DESCRIPTION
**Pull request by be1ski**

### CODE REVIEW

**Comment by bicenekin** [comment]
Hi!
Can you update the "Manifest" for android 12 and SDK API 31

File: app/build.gradle [hunk 1]
```diff
@@ -41,5 +41,5 @@ android {
 }
 
 dependencies {
-    implementation ('com.travelpayouts:travel-sdk:1.4.2')
+    implementation ('com.travelpayouts:library-travel-sdk:2.0.0')
```
Comment (be1ski) [inline app/build.gradle:44]:
```suggestion
    implementation ('com.travelpayouts:library-travel-sdk:2.0.0-rc1')
```

File: app/build.gradle [hunk 2]
```diff
@@ -41,5 +41,5 @@ android {
 }
 
 dependencies {
-    implementation ('com.travelpayouts:travel-sdk:1.4.2')
+    implementation ('com.travelpayouts:library-travel-sdk:2.0.0-rc1')
```
Comment (be1ski) [inline app/build.gradle:44]:
```suggestion
    implementation ('com.travelpayouts:library-travel-sdk:2.0.0-rc2')
```

**Comment by arya3073** [comment]
Sir you can change in sdk to make public in google play console. Because Activity is not exported in it.

**Comment by bicenekin** [comment]
Thank you for the update above @be1ski 
We have another issue after latest updates in google play with webView SS'd below. We assume it's for Car Rental program. Can you pls check it.
Screenshot 2023-03-01 at 13 36 07

**Comment by arya3073** [comment]
One more problem can be seen in this that the logos of the agencies of the winning flights are not being shown. Because it used to be like in the previous update.

**Comment by arya3073** [comment]
Sir, @be1ski  there is a problem in hotel search, city or hotel name is not being searched.

**Review by Object173 — APPROVED** [review]

**Pull request merged**